tools/marvin/setup.py

```diff
@@ -27,7 +27,7 @@
         raise RuntimeError("python setuptools is required to build Marvin")
 
 
-VERSION = "4.21.0.0-SNAPSHOT"
+VERSION = "4.21.0.0"
```

Comment:
don’t think we want to merge this bit, yet.

Reply:
reverted